### PR TITLE
[3.9] HTML tags are not recognized in tooltip

### PR DESCRIPTION
**Pull request by Kostelano**
Pull Request for Issue # .

### Summary of Changes
Minor fix for tooltip in user profile (see screenshots).

### Before / After
Screenshot_3

### Before / After - Hathor template
Screenshot_1

### CODE REVIEW

**Comment by brianteeman** [comment]
Since April 2017

> We strongly recommend all users to use the Isis template. The Hathor template has not been updated to work with the new features shipping with this release.
When we released Hathor it was build with accessibility in mind, but over the years the Isis template has been improved in terms of accessibility and supporting technologies. We now believe that the Isis template is as good as or possibly better than Hathor.
To ensure a smooth experience for our users, during the update process, the default administrator template will be switched to Isis. You are free to switch this back after the update but we cannot guarantee and in fact know that many features will not work with Hathor.

**Comment by infograf768** [comment]
This solution has the advantage of not modifying the lang strings.
But, normally, we should use a sprintf and modify the label string to include a br and a variable which would load the desc.
This Pr is not only for hathor...

**Comment by Quy** [comment]
or convert it to be a popover to be consistent with other fields.

28771-popover

**Comment by infograf768** [comment]
hmm
at this time, using isis or Hathor, we do have correctly
Screen Shot 2020-04-24 at 07 44 42
Screen Shot 2020-04-24 at 07 47 59

Firefox Macintosh.

**Comment by infograf768** [comment]
But, as @Quy says we can use popover to normalise with other tips
```
			<div class="control-label">
				<?php JHtml::_('bootstrap.popover');
					$text        = JText::_('COM_USERS_USER_FIELD_TWOFACTOR_LABEL');
					$description = JText::_('COM_USERS_USER_FIELD_TWOFACTOR_DESC');
					$position    = null;
					$title       = ' title="' . htmlspecialchars(trim($text, ':')) . '"'
						. ' data-content="'. htmlspecialchars($description) . '"';

					if (JFactory::getLanguage()->isRtl())
					{
						$position = ' data-placement="left" ';
					}
				?>
				<label id="jform_twofactor_method-lbl" for="jform_twofactor_method" class="hasPopover"
						<?php echo $title; ?><?php echo $position; ?>>
					<?php echo JText::_('COM_USERS_USER_FIELD_TWOFACTOR_LABEL'); ?>
				</label>
			</div>
```

**Comment by infograf768** [comment]
@Kostelano 
Are you going to update your PR with the code I provided or shall we close this and I will make a new PR?

**Comment by Kostelano** [comment]
Yes of course. I apologize for not being able to answer earlier.
I changed this according to your recommendation.
Everything works, thank you.

Screenshot_1

File: administrator/components/com_users/views/user/tmpl/edit.php [hunk 1]
```diff
@@ -82,11 +82,23 @@
...
-				<label id="jform_twofactor_method-lbl" for="jform_twofactor_method" class="hasTooltip"
-						title="<?php echo '<strong>' . JText::_('COM_USERS_USER_FIELD_TWOFACTOR_LABEL') . '</strong><br />' . JText::_('COM_USERS_USER_FIELD_TWOFACTOR_DESC'); ?>">
-					<?php echo JText::_('COM_USERS_USER_FIELD_TWOFACTOR_LABEL'); ?>
-				</label>
+			<div class="control-label">
+				<?php JHtml::_('bootstrap.popover');
+					$text        = JText::_('COM_USERS_USER_FIELD_TWOFACTOR_LABEL');
+					$description = JText::_('COM_USERS_USER_FIELD_TWOFACTOR_DESC');
+					$position    = null;
+					$title       = ' title="' . htmlspecialchars(trim($text, ':')) . '"'
```
Comment (Quy) [inline administrator/components/com_users/views/user/tmpl/edit.php:90]: Not necessary to apply `htmlspecialchars` when it doesn't contain user input text. Why trim `:`?

Comment (infograf768) [reply administrator/components/com_users/views/user/tmpl/edit.php:90]: I just copied the code from
https://github.com/joomla/joomla-cms/blob/staging/layouts/joomla/form/renderlabel.php

similar code elsewhere for privacyconsent, etc.

**Comment by infograf768** [comment]
There are issues with wrong end of line and a concat code style

**Pull request closed without merging**